modules/osdk-upgrading-v130-to-v180.adoc

```diff
@@ -0,0 +1,895 @@
+
+... Create the `ServiceAccount` definition in a file:
++
+.`config/rbac/service_account.yaml` file
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: controller-manager
+  namespace: system
+----
+====
+
+... Add the service account to the list of RBAC resources:
++
+[source,terminal]
+----
+$ echo "- service_account.yaml" >> config/rbac/kustomization.yaml
+----
+
+... Update all `RoleBinding` and `ClusterRoleBinding` objects that reference the Operator's service account:
++
+[source,terminal]
+----
+$ find config/rbac -name *_binding.yaml -exec sed -i -E 's/  name: default/  name: controller-manager/g' {} \;
+----
+
+... Add the service account name to the manager deployment's `spec.template.spec.serviceAccountName` field:
++
+[source,terminal]
+----
+$ sed -i -E 's/([ ]+)(terminationGracePeriodSeconds:)/\1serviceAccountName: controller-manager\n\1\2/g' config/manager/manager.yaml
+----
+
+... Verify the changes look like the following diffs:
++
+.`config/manager/manager.yaml` file diff
+[%collapsible]
+====
+[source,diff]
+----
+...
+           requests:
+             cpu: 100m
+             memory: 20Mi
++      serviceAccountName: controller-manager
+       terminationGracePeriodSeconds: 10
+----
+====
++
+.`config/rbac/auth_proxy_role_binding.yaml` file diff
+[%collapsible]
+====
+[source,diff]
+----
+...
+   name: proxy-role
+ subjects:
+ - kind: ServiceAccount
+-  name: default
++  name: controller-manager
+   namespace: system
+----
+====
++
+.`config/rbac/kustomization.yaml` file diff
+[%collapsible]
+====
+[source,diff]
+----
+ resources:
++- service_account.yaml
+ - role.yaml
+ - role_binding.yaml
+ - leader_election_role.yaml
+----
+====
++
+.`config/rbac/leader_election_role_binding.yaml` file diff
+[%collapsible]
+====
+[source,diff]
+----
+...
+   name: leader-election-role
+ subjects:
+ - kind: ServiceAccount
+-  name: default
++  name: controller-manager
+   namespace: system
+----
+====
++
+.`config/rbac/role_binding.yaml` file diff
+[%collapsible]
+====
+[source,diff]
+----
+...
+   name: manager-role
+ subjects:
+ - kind: ServiceAccount
+-  name: default
++  name: controller-manager
+   namespace: system
+----
+====
++
+.`config/rbac/service_account.yaml` file diff
+[%collapsible]
+====
+[source,diff]
+----
++apiVersion: v1
++kind: ServiceAccount
++metadata:
++  name: controller-manager
++  namespace: system
+----
+====
+
+. Make the following changes to your `config/manifests/kustomization.yaml` file:
+
+.. Add a link:https://kustomize.io/[Kustomize] patch to remove the link:https://cert-manager.io/[cert-manager] `volume` and `volumeMount` objects from your cluster service version (CSV).
++
+Operator Lifecycle Manager (OLM) does not yet support cert-manager, so a JSON patch was added to remove this volume and mount so OLM can itself create and manage certificates for your Operator.
```

Comment:
up to you: "...so OLM can create and manage certificates itself for your Operator."

Or maybe no "itself" at all?

Reply:
Went with: "Because Operator Lifecycle Manager (OLM) does not yet support cert-manager, a JSON patch was added to remove this volume and mount so OLM can create and manage certificates for your Operator."